WordPress/Classes/ViewRelated/Reader/Detail/Views/ReaderDetailFeaturedImageView.swift

```diff
@@ -288,7 +288,11 @@ class ReaderDetailFeaturedImageView: UIView, NibLoadable {
 
         if let scrollView = self.scrollView {
             scrollView.contentInset = UIEdgeInsets(top: height, left: 0, bottom: 0, right: 0)
-            scrollView.setContentOffset(CGPoint(x: 0, y: -height), animated: false)
+
+            // Only reset offset if the user has scrolled just a little bit
```

Comment:
I'm not sure what "just a little bit" means in this context. Can we change it something a bit more descriptive?